bundles/ApplicationLoggerBundle/config/services.yaml

```diff
@@ -27,13 +26,11 @@ services:
     # mimics functionality from the ApplicationLogger itself to be used from any
     # monolog handler.
     Pimcore\Bundle\ApplicationLoggerBundle\Processor\ApplicationLoggerProcessor:
-        public: false
```

Comment:
Add `~` to the service.

Reply:
must have missed that - thx 
https://github.com/pimcore/pimcore/pull/14797/commits/6a8d1d16d8ef0e0c33f8474c8a03c110a64dad8c